code/script.js

```diff
@@ -1,47 +1,120 @@
-    chat.innerHTML += `
-      <section class="user-msg">
-        <div class="bubble user-bubble">
-          <p>${message}</p>
-        </div>
-        <img src="assets/user.png" alt="User" />  
-      </section>
-    `
-  } else if (sender === 'bot') {
-    chat.innerHTML += `
-      <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
-        <div class="bubble bot-bubble">
-          <p>${message}</p>
-        </div>
-      </section>
-    `
-  }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
-}
-
-// Starts here
-const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
-
-// Set up your eventlisteners here
-
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000)
+    const showMessage = (message, sender) => {
+      if (sender === 'user') {
+        chat.innerHTML += ` 
+          <section class="user-msg">
+            <div class="bubble user-bubble">
+              <p>${message}</p>
+            </div>
+            <img src="./assets/user.png" alt="User" />  
+          </section>
+        `
+
+      } else if (sender === 'bot') {
+        chat.innerHTML += ` 
+          <section class="bot-msg">
+          <img src="./assets/sushi-roll-icon.jpeg" alt="">
+            <div class="bubble bot-bubble">
+              <p>${message}</p>
+            </div>
+          </section>
+        `
+      chat.scrollTop = chat.scrollHeight
+    }
+ }
+
+//5.
+    const showFoodOptions = () => {
+      currentQuestion++ // current question is 2
+      setTimeout(() => showMessage(`What sushi would you like to order? `, 'bot'), 1000)
+      inputWrapper.innerHTML = `
+        <button id="small-sushi">8 pieces</button>
+        <button id="big-sushi">12 pieces</button>
+      `
+
+      const smallSushi = document.getElementById('small-sushi')
+      smallSushi.addEventListener('click', () => nextQuestion('8 pieces'))
+
+      const bigSushi = document.getElementById('big-sushi')
+      bigSushi.addEventListener('click', () => nextQuestion('12 pieces'))
+     
+    }
+// 7.
+    const showPriceSushi = () => {
+      currentQuestion++// current question is 3
+      inputWrapper.innerHTML = `
+        <button id="yes">Yes</button>
+        <button id="no">No</button>
+      ` 
+      const yesAnswer = document.getElementById('yes'); 
+      yesAnswer.addEventListener('click', () => nextQuestion('yes'));
+
+      const noAnswer = document.getElementById('no'); 
+      noAnswer.addEventListener('click', () => nextQuestion('no'));
```

Comment:
The function when I click "NO" and chat restarts is good. If you had more time maybe the bot could have answered "Okay I understand, see you next time" and then restart?

Reply:
Yeah, that's a great idea! When I have time I am gonna add it :)

Reply:
Yes, great idea!